Makefile

```diff
@@ -84,6 +84,19 @@ revive: scripts/go/bin/revive
 		-config ./scripts/go/configs/revive.toml \
 		$(GO_FILES)
 
+# create docker-compose file with provided sources and start them
+# example: make env sources=postgres,openldap
+env: env-down
```

Comment:
I wonder if `env` is a good name. Would `devenv` work or would that conflict with the folder name? I need to read up on makefiles :)

Reply:
We can define it as `devenv`! We would just have defined as `PHONY` :)

Reply:
Then I think we should go with `devenv` since `env` is a common word that can have many different meanings.

Reply:
Done!